modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+OpenShift API for Data Protection (OADP) is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate data from one {product-title} cluster to another or to migrate data from a VMware cluster to an {product-title} cluster.
+
+You can use OADP to back application data from one cluster and restore it on another. However, doing so is both more complicated than using {mtc-short} or using OADP to back up and restore to the same {product-name} cluster.
+
+== Considerations in using OADP to back up application data from one cluster and restore it to another
+
+Before you begin planning to use OADP to back up application data from one cluster and restore it to another cluster, consider the following limitations:
+
+* You must exclude Operators from backup and restore operations because OADP is not designed to either back up or restore Operators.
+* Velero, which OADP is built upon, does not natively support the migration of persistent volumes snapshots across cloud providers. To migrate volume data between cloud platforms, you must enable Velero's File System Backup, which backs up volume contents at the filesystem level.
+* You also must use Velero's File System Backup if you want to migrate data between regions on AWS or Azure.
+* Velero does not support restoring data to a cluster with a lower Kubernetes version than the source cluster.
+* It is theoretically possible to migrate workloads to a destination with a higher Kubernetes version than the source, but you must consider the compatibility of API groups between clusters for each custom resource. If a Kubernetes version upgrade breaks the compatibility of core or native API groups,you must first update the impacted custom resources. For more information about API group versions, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/backup_and_restore/index#oadp-different-kubernetes-api-versions[Working with different Kubernetes API versions on the same cluster].
+* The UID/GID range of the namespace might change on the destination cluster. OADP does not backup and restore OpenShift UID/GID range metadata. If the backed application requires a specific UUID, ensure the range is available when restored. For more information on OpenShift's UID/GID range link:https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids[A Guide to OpenShift and UIDs]. Allowing OADP to create the namespace while restoring may avoid the issue.
```

Comment:
> Allowing OADP to create the namespace while restoring may avoid the issue.

I am not aware we are doing anything with namespace UID today.. have to double check.

Reply:
todo: update faq

Reply:
https://access.redhat.com/articles/5456281#issues-restoring-an-oadp-backup-application-unable-to-access-data-23 has been updated.

Reply:
Pending another validation that letting velero create the namespace on restore works (keeps annotation from backup).. will make another faq update and request changes here.

Reply:
validated that the namespace’s annotated scc uid-ranges are restored properly on oadp.v1.1.4 across cluster with openshift,aws,csi plugins enabled. Updating FAQ again.